dev/ProjectReunion_DLL/ProjectReunion_DLL.vcxproj

```diff
@@ -406,4 +407,4 @@
   <Target Name="CopyFiles" AfterTargets="AfterBuild">
     <Copy SourceFiles="@(PublicHeaders)" DestinationFolder="$(OutDir)" />
   </Target>
-</Project>
+</Project>
```

Comment:
NIT: github's cranky there's no newline at the end of the file. A sideeffect of your editor?

Reply:
It must be, looks like this happens to my other files too.